samples/snippets/create_custom_entry.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def create_custom_entry(override_values):
+    """Creates a custom entry within an entry group."""
+    # [START data_catalog_create_custom_entry]
+    # Import required modules.
+    from google.cloud import datacatalog_v1
+
+    # Google Cloud Platform project.
+    project_id = "my-project"
+    # Entry group to be created.
+    entry_group_id = "my_entry_group"
+    # Entry to be created.
+    entry_id = "my_entry_id"
+    # Tag Template to be created.
+    tag_template_id = "my_tag_template"
+    # Tag to be created.
+    tag_name = "my_cool_tag"
+
+    # [END data_catalog_create_custom_entry]
+
+    # To facilitate testing, we replace values with alternatives
+    # provided by the testing harness.
+    project_id = override_values.get("project_id", project_id)
+    entry_group_id = override_values.get("entry_group_id", entry_group_id)
+    entry_id = override_values.get("entry_id", entry_id)
+    tag_template_id = override_values.get("tag_template_id", tag_template_id)
+    tag_name = override_values.get("tag_name", tag_name)
+
+    # [START data_catalog_create_custom_entry]
+    # For all regions available, see:
+    # https://cloud.google.com/data-catalog/docs/concepts/regions
+    location = "us-central1"
+
+    # Use Application Default Credentials to create a new
```

Comment:
Is this standard boilerplate for the python samples these days?  Can we point readers towards ADC info page, as this info could be misleading due to brevity.

Reply:
No, I don't think this is the current boilerplate. Looks like it's good to just remove it.